src/InputForm.js

```diff
@@ -22,6 +23,9 @@ class InputForm extends Component {
             this.setState({
                 locationObject: response.data[0]
             })
+            this.setState({

```

Comment:
This is a HUGE help to me to be able to see a working version. I wasn't able to get mine to work because I wasn't entering in the ImageURL correctly.